src/frontmatter/frontmatter.ts

```diff
@@ -1,26 +1,20 @@
 import * as yaml from 'js-yaml';
-import _get = require('lodash/get');
-import _pullAt = require('lodash/pullAt');
-import _set = require('lodash/set');
-import _toPath = require('lodash/toPath');
-import _unset = require('lodash/unset');
+import { get, pullAt, set, toPath, unset } from 'lodash';
```

Comment:
Will this be tree-shaken?

Reply:
I've tested it w babel, which we are using in dojo app and I'm introducing to studio build process, and yes it is tree shaken :). However, babel does not support the weird `import x = require(y);` type syntax, so I've been slowly removing that from our various repos.

Reply:
> I've tested it w babel, which we are using in dojo app and I'm introducing to studio build process, and yes it is tree shaken :)

Sounds good, let's merge it in!

> import x = require(y);

Yeah, it's TS-specific.